src/main/java/prblm/Circle.java

```diff
@@ -0,0 +1,22 @@
+package prblm;
+
+public class Circle implements Shape {
+    private final double radius;
+    final double pi = Math.PI;
+
+    public Circle() {
```

Comment:
why would you add this? There is no point to have a deafault constructor. And btw I saw this to one of your colleagues as well.